### PR TITLE
Citizen Dashboard View

### PR DESCRIPTION
**Pull request by mehsheed**

### CODE REVIEW

File: SIMPLE/src/edu/manipal/icas/simple/controllers/RouteController.java [hunk 1]
```diff
@@ -64,6 +65,10 @@ public void routeTo(Route route) {
 		case OFFICER_LOGIN:
 			displayView(new OfficerLoginViewImpl());
 			break;
+		case CITIZEN_DASHBOARD:
+			displayView(new CitizenDashboardViewImpl());
+			
+
```
Comment (Vishwas-Adiga) [inline SIMPLE/src/edu/manipal/icas/simple/controllers/RouteController.java:71]: Fix whitespace please

File: SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenDashboardViewImpl.java [hunk 2]
```diff
@@ -0,0 +1,253 @@
+/**
+ * 
+ */
```
Comment (Vishwas-Adiga) [inline SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenDashboardViewImpl.java:3]: Remove comment?

File: SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenDashboardViewImpl.java [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+import javax.swing.JTable;
+import edu.manipal.icas.simple.utils.ResourceConstants;
+import edu.manipal.icas.simple.views.CitizenDashboardView;
+import net.miginfocom.swing.MigLayout;
+
+/**
+ * @author Mehsheed(syed.ahmed2@learner.manipal.edu)
+ *
+ * 
+ */
```
Comment (Vishwas-Adiga) [inline SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenDashboardViewImpl.java:21]: Please add a doc line

File: SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenDashboardViewImpl.java [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+     			{"Have you ever surrendered your Indian passpor:",""},
+     			{"Have you ever applied for renunciation of Indian citizenship:",""},
+     			{"Have you ever returned to India on Emergency Certificate (EC):",""},
+     			{"Have you ever been deported from any country:",""},
+     			{"Have you ever been repatriated from any country back to India:",""}
+     	};
+	
+	
+	
+
```
Comment (Vishwas-Adiga) [inline SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenDashboardViewImpl.java:66]: Whitespace.
Do Ctrl+Shift+F on all committed files

File: SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenDashboardViewImpl.java [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+		
+		
+		
+	}
+	
+	
+	
+	/**
+	 * 
+	 */
```
Comment (Vishwas-Adiga) [inline SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenDashboardViewImpl.java:87]: Remove or add javadoc?

File: SIMPLE/src/edu/manipal/icas/simple/views/CitizenDashboardView.java [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+ *
+ * 
+ */
+public interface CitizenDashboardView extends View {
+	/**
+	 * Gets the button that the citizen clicks on to start application for a new passport.
+	 * 
+	 * @return Apply for new passport button
+	 */
+	JButton getAppForNewPPButton();
```
Comment (Vishwas-Adiga) [inline SIMPLE/src/edu/manipal/icas/simple/views/CitizenDashboardView.java:21]: Please use full name, i.e., getApplicationForNewPassportButton

File: SIMPLE/src/edu/manipal/icas/simple/views/CitizenDashboardView.java [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+	/**
+	 * Gets the button that the citizen clicks on to start application for Re-Issue of a passport.
+	 * 
+	 * @return Apply for re-issue passport button
+	 */
+	
+	
+	JButton getredirect2AppstepsButton();
```
Comment (Vishwas-Adiga) [inline SIMPLE/src/edu/manipal/icas/simple/views/CitizenDashboardView.java:29]: Pascal casing, and please use full name (Two instead of 2)

**Review by Vishwas-Adiga — CHANGES_REQUESTED** [review]

**Comment by Rea-Mammen** [comment]
Could you please add an icon for the frame? 
https://github.com/ICS-233-SIMPLE/simple-java/blob/2c04c4fb1115efbccd1b06ceb58d3f8b74b50484/SIMPLE/src/edu/manipal/icas/simple/impl/views/CitizenLoginViewImpl.java#L158
Use this as ref

**Review by Yodhin05 — APPROVED** [review]

File: SIMPLE/.classpath [hunk 8]
```diff
@@ -15,5 +15,6 @@
 	<classpathentry kind="lib" path="lib/commons-io-2.11.0.jar"/>
 	<classpathentry kind="lib" path="lib/icepdf-core-6.2.2.jar"/>
 	<classpathentry kind="lib" path="lib/icepdf-viewer-6.2.2.jar"/>
+	<classpathentry kind="lib" path="C:/Users/sameh/Documents/GitHub/Route/simple-java/SIMPLE/lib/miglayout-4.0.jar"/>
```
Comment (Vishwas-Adiga) [inline SIMPLE/.classpath:18]: I think you used "add external jar" in Eclipse. This classpathentry won't work on our machines. Can you change it to the ones listed in other lines?

**Review by Vishwas-Adiga — APPROVED** [review]
Looks good!

**Review by Rea-Mammen — APPROVED** [review]

**Pull request merged**